Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,86 @@
+| (Cinder) Volume    | IaaS resource representing block storage disk that can be attached as a virtual disk to virtual machines. Managed by the Cinder service. |
+| (Volume) Snapshot  | Thinly-provisioned copy-on-write snapshots of volumes. Stored in the same Cinder storage backend as volumes.                             |
+| Volume Type        | Attribute of volumes determining storage details of a volume such as backend location or whether the volume will be encrypted.           |
+| (Barbican) Secret  | IaaS resource storing cryptographic assets such as encryption keys. Managed by the Barbican service.                                     |
+| Key Encryption Key | IaaS resource, used to encrypt other keys to be able to store them encrypted in a database.                                              |
+| Floating IP        | IaaS resource, an IP that is usually routed and accessible from external networks.                                                       |
+| Disk               | A physical disk drive (e.g. HDD, SSD) in the infrastructure.                                                                             |
+| Node               | A physical machine in the infrastructure.                                                                                                |
+| Cyber threat       | Attacks on the infrastructure through the means of electronic access.                                                                    |
+
+## Context
+
+Some standards provided by the SCS project will talk about or require procedures to backup resources or have redundancy for resources.
+This decision record should discuss, which failure threats are CSP-facing and will classify them into several levels.
+In consequence these levels should be used in standards concerning redundancy or failure-safety.
+
+## Decision
+
+First there needs to be an overview about possible failure cases in infrastructures:
+
+| Failure Case | Probability | Consequences |
+|----|-----|----|
+| Disk Failure/Loss | High | Data loss on this disk. Impact depends on type of lost data (data base, user data) |
+| Node Outage | Medium to High | Data loss on node / (temporary) loss of functionality and connectivity of node (impact depends on type of node)  |
+| Rack Outage | Medium | similar to Disk Failure and Node Outage |
+| Power Outage (Data Center supply)  | Medium | potential data loss, temporary loss of functionality and connectivity of node (impact depends on type of node)  |
+| Fire | Medium | permanent Disk and Node loss in the affected zone |
+| Flood | Low | permanent Disk and Node loss in the affected zone |
+| Earthquake | Very Low | permanent Disk and Node loss in the affected zone |
+| Storm/Tornado | Low | permanent Disk and Node loss in the affected fire zone |
+| Cyber threat  | High | permanent loss or compromise of data on affected Disk and Node  |
+
+These failure cases can result in temporary (T) or permanent (P) loss of the resource or data within.
```

Comment:
I like this table. It is a very good starting point to think about backup and failure strategies. IMO, its content is not in a final state. As well as the following paragraphs. They look a bit incomplete. We should take an other round on brainstorming to take OpenStack services and k8s resources and services into account as well. 

At the end, I like to see an overview on how use cases defined in upper table effect availability of all OpenStack and k8s resources and services. This overview MUST include all fail-safe strategies SCS project requires together with the use case they prevent. Maybe this overview would a separate standard or references other standards. We have to discuss this.